Assets/Spawner/Spawner.cs

```diff
@@ -617,13 +770,19 @@ public override IList<Wave> GetChildren() {
     public override bool AddWave(int index, Wave wave) {
       return false;
     }
+
+    public override void GetSpawnTimes(ref EnemySpawnTimes spawnTimes, ref float projectedStartTime) {
+      if (Finished) return;
```

Comment:
this is missing something.
Say we have a 30 second spacer wave and we are 15 seconds through it.
this will return that the next wave is delay in the future but it should be 15 seconds.

Reply:
I chjanged the projectedStartTime modification to account for this situation.